apps/api/lib/api/web/controllers/user_controller.ex

```diff
@@ -46,6 +46,12 @@ defmodule API.Web.UserController do
     end
   end
 
+  def unauthenticated(conn, _params) do
```

Comment:
Im not sure this belongs in this module, since its going to be used for many potential controllers. Maybe we can throw it into a generic Guardian module? We already have the Guardian Serializer module that maybe we can name more generically?